classes/asserters/adapter/call.php

```diff
@@ -38,6 +38,7 @@ public function __get($property)
             case 'atleastonce':
             case 'wascalled':
             case 'wasnotcalled':
+            case 'notreceiveanymessage':
```

Comment:
il faudrait pas rajouter une méthode dans cette class pour ceci aussi?

Reply:
if we want to be correct with the english language and the logic of « wasNotCalled » we should name this method « didNotReceiveAnyMessage »